privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -216,15 +216,15 @@ angular.module("privacyideaApp")
 
                 // Challenge Response always containes mult_challenge!
                 var multi_challenge = error.detail.multi_challenge;
-                if (multi_challenge.length > 1) {
+                if (multi_challenge.length >= 1) {
                     $scope.challenge_message = gettextCatalog.getString('Please confirm with one of these tokens:');
```

Comment:
This is not how it is ment originially. If there is only *one* challenge token, then the message of this token should be displayed like:

> please enter the OTP value from your SMS

and not

> Please confirem with on of these tokens: SMS001012

Am I getting something wrong here?

Reply:
Problem is that your dont get first message